crates/core/app/tests/common/mod.rs

```diff
@@ -47,7 +51,7 @@ pub async fn start_test_node(storage: &TempStorage) -> anyhow::Result<PenumbraTe
     let app_state = AppState::default();
     let consensus = Consensus::new(storage.as_ref().clone());
     TestNode::builder()
-        .single_validator()
+        .with_penumbra_single_validator()
         .with_penumbra_auto_app_state(app_state)?
```

Comment:
What's the relation between these two methods?

Reply:
xref https://github.com/penumbra-zone/penumbra/pull/3902#issuecomment-1971674791

Reply:
xref https://github.com/penumbra-zone/penumbra/pull/3902#issuecomment-1972239643

we talked this through and found a better way to do this!